doc/nrf/glossary.rst

```diff
@@ -635,9 +677,19 @@ Glossary
    Target
       The goal of an operation, for example, programming a specific image on a device, compiling a specific set of files, or removing previously generated files.
 
+   Target Wake Time (TWT)
+      A function that allows an access point to define a specific time or set of times for individual devices to access the wireless network.
+      It is the mechanism that allows devices to negotiate when and how often they turn on and off, based on when they need to send and receive data.
+
+   Temporal Key Integrity Protocol (TKIP)
+      A security protocol used in the IEEE 802.11 wireless networking standard.
+
    Time to First Fix (TTFF)
       The time needed by a :term:`Global Navigation Satellite System (GNSS)` module to estimate its position.
 
+   Traffic Indication Map (TIM)
+      A structure used in 802.11 wireless network management frames.
```

Comment:
@bama-nordic do we need to explain a bit more on wha TIM is used for?

Reply:
Same as we explained for DTIM

Reply:
@udaynordic